python/lsst/daf/butler/formatters/logs.py

```diff
@@ -0,0 +1,121 @@
+# This file is part of daf_butler.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+__all__ = ("ButlerLogRecordsFormatter", )
+
+from typing import (
+    Any,
+    Optional,
+    Type,
+)
+
+from lsst.daf.butler import ButlerLogRecords, ButlerLogRecord
+from .json import JsonFormatter
+
+
+class ButlerLogRecordsFormatter(JsonFormatter):
+    """Read and write log records in JSON format.
+
+    This is a naive implementation that treats everything as a pydantic.
+    model.  In the future this may be changed to be able to read
+    ``ButlerLogRecord`` one at time from the file and return a subset
+    of records given some filtering parameters.
+    """
+
+    def _readFile(self, path: str, pytype: Optional[Type[Any]] = None) -> Any:
+        """Read a file from the path in JSON format.
+
+        Parameters
+        ----------
+        path : `str`
+            Path to use to open JSON format file.
+        pytype : `class`, optional
+            Not used by this implementation.
```

Comment:
Is this true?

Reply:
No. I could not use it but since it's there I am using it. Thanks.